### PR TITLE
feat: add type column to WebLogView

### PR DESCRIPTION
**Pull request by cristianoventura**
## Description

This PR adds a new `Type` column to the `WebLogView` component to differentiate static assets from dynamic requests.

## How to Test

- Toggle the "Show static assets" switch
- Check that the Type column shows the appropriate label for each request

## Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [x] I have run linter locally (`npm run lint`) and all checks pass.
- [ ] I have run tests locally (`npm test`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

## Screenshots (if appropriate):

image

## Related PR(s)/Issue(s)

Resolves https://github.com/grafana/k6-studio/issues/325

### CODE REVIEW

**Review by e-fisher — APPROVED** [review]
LGTM

File: src/components/WebLogView/Row.tsx [hunk 1]
```diff
@@ -53,6 +54,11 @@ export function Row({ data, isSelected, onSelectRequest }: RowProps) {
           />
         </ResponseStatusBadge>
       </Table.Cell>
+      <Table.Cell>
+        <Text size="1">
+          {isNonStaticAssetResponse(data) ? 'Dynamic' : 'Static'}
```
Comment (going-confetti) [inline src/components/WebLogView/Row.tsx:59]: I was under the impression we wanted this to work similarly to the browser dev tools:
image
image
If we only want to mark static requests, I suggest something that will take less horizontal space in `WebLogView` - maybe an icon or something like that

File: src/components/WebLogView/WebLogView.utils.ts [hunk 2]
```diff
@@ -20,3 +22,12 @@ export function isGroupedProxyData(
 ): data is GroupedProxyData {
   return !Array.isArray(data)
 }
+
+export function getRequestType(data: ProxyData) {
+  if (isNonStaticAssetResponse(data)) {
+    return 'fetch'
```
Comment (cristianoventura) [inline src/components/WebLogView/WebLogView.utils.ts:28]: ChromeDevTools categorizes this a bit more. I was debating whether to call it "document", "dynamic" or "fetch". Any thoughts?

**Review by cristianoventura — COMMENTED** [review]
@going-confetti I made a change to include a more descriptive "type" based on the content-type returned by the request.

image

File: src/components/WebLogView/WebLogView.utils.ts (hunk 2 above)
Comment (going-confetti) [reply src/components/WebLogView/WebLogView.utils.ts:28]: Perhaps, we can make use of the content-type header to categorize different types of dynamic requests?

**Review by going-confetti — APPROVED** [review]

**Pull request merged**